### DISCUSSION TITLE
[ARCHIVED] Exercise - Starter Page

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#21] [Homework]
Create a new HTML page, including at least:

- one heading, one [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)
- one paragraph 
- one image.

Check our the [video tutorial](https://www.youtube.com/watch?v=STPUv_f1IIw&list=PLyQtZRmg0ogeK2CitMK4fiQa8228_rxiV&index=3) if needed. Post a screenshot of your webpage as a comment!

**Comment by RobbinHezhengWu** [comment]
Robbin Hezheng Wu's homework
![Screen Shot](https://github.com/periode/augmenting-gallery/assets/144832282/862dab4e-03f4-43e3-9ddb-15f93a5e9e2b)

**Comment by Caythian** [comment]
<img width="1470" alt="Screenshot 2023-09-12 at 8 39 28 PM" src="https://github.com/periode/augmenting-gallery/assets/140059888/b1a61186-9851-4449-b9b2-2ab63eb4a7d4">
-Catherine Wang

**Comment by bhornik** [comment]
<img width="1511" alt="Screenshot 2023-09-12 at 9 47 32 PM" src="https://github.com/periode/augmenting-gallery/assets/99111311/dc0acbd4-a97b-498c-8af9-bb13bdc62994">
- Beckett Hornik

**Comment by zbyiii** [comment]
<img width="1431" alt="Bingyi Zhang Website" src="https://github.com/periode/augmenting-gallery/assets/144227589/b0abd36c-dfbf-4f9f-b3ec-424873fc5ebb">

**Comment by lizalmonds** [comment]
Liz's Exercise 1:

![image](https://github.com/periode/augmenting-gallery/assets/144227299/e279b060-dce3-449d-95f5-be5411aaad31)

**Comment by Ambraaaliu** [comment]
<img width="1512" alt="截圖 2023-09-13 00 44 31" src="https://github.com/periode/augmenting-gallery/assets/131328084/2f94a206-fe84-4d36-9dbb-92268c8c855f">

**Reply by Ambraaaliu** [reply]
https://ambraaaliu.github.io/AugmentingGallery/AmbrasDoodlePage/

**Comment by liunikolai** [comment]
<img width="1440" alt="Screenshot 2023-09-13 at 2 58 11 AM" src="https://github.com/periode/augmenting-gallery/assets/144227273/31987e21-408e-49c7-85f8-94680c581b11">

**Comment by chaerin-d** [comment]
[file:///Users/chaerindo/Desktop/__/23-24%20NYU/23%20Fall/AG/SSAM.html](url)

**Comment by jacksonkkingg** [comment]
Sorry I didn't see this page! I didn't realize I had to upload this. 
<img width="1432" alt="Screen Shot 2023-09-20 at 7 40 34 AM" src="https://github.com/periode/augmenting-gallery/assets/132478506/cdd77081-66c5-40b0-8f58-502778de2752">

[](url)
